src/server/routes/plywood/plywood.ts

```diff
@@ -97,10 +97,12 @@ export function plywoodRouter(settingsManager: Pick<SettingsManager, "anchorPath
       };
       res.json(reply);
     } catch (error) {
-      logger.log("error:", error.message);
+      let message = error.message;
```

Comment:
I think moving this logic of extracting the message from error, checking if it has the `stack` property etc. to a helper function could clean up the code a little bit